### PR TITLE
fix(suite-native): use correct message config on mobile

### PR DESCRIPTION
**Pull request by vytick**
I realised that we dont properly differentiate between prod and develop on mobile while fetching the message config. The app does not use process.env so I had to update env checking.

@matejkriz Please, check the logic behind choosing the right url for Suite 🙏

### CODE REVIEW

**Review by juriczech — APPROVED** [review]

File: suite-native/app/.env.develop [hunk 1]
```diff
@@ -3,4 +3,5 @@ COMMIT_HASH=
 CHANGELOG=
 VERSION=
 BUILD_NUMBER=
+CODESIGN_BUILD=true
```
Comment (matejkriz) [inline suite-native/app/.env.develop]: I would expect `develop` and maybe even `staging` environment to have `CODESIGN_BUILD=false`, how else could QA look at a new message-system config before releasing it to production?

Comment (vytick) [reply suite-native/app/.env.develop]: Good spot! 😬 That is an error left from testing. fixed in 97296bcbe0

File: suite-native/message-system/src/components/FeatureMessageScreen.tsx [hunk 2]
```diff
@@ -126,7 +126,7 @@ export const FeatureMessageScreen = () => {
                     </Button>
                 )}
                 {isDismissable && (
-                    <Button size="large" onPress={handleDismiss}>
+                    <Button size="large" colorScheme="tertiaryElevation0" onPress={handleDismiss}>
```
Comment (juriczech) [inline suite-native/message-system/src/components/FeatureMessageScreen.tsx:129]: According to design, this is wrong. Why did you change this?

image

**Review by matejkriz — COMMENTED** [review]
Now we use `codesign` as one word everywhere in the code https://github.com/trezor/trezor-suite/blob/develop/suite-common/suite-utils/src/build.ts/#L3 Both versions are OK, I guess, but we should use it consistently.

When adding this util we should refactor the usage in the web/desktop Suite so we don't use three ways (we already use `IS_CODESIGN_BUILD` for the build and only `CODESIGN_BUILD` for the application code).

Do you have any plan on how to deal with app versions that have already been released to production and are looking at develop message-system config? Maybe after some time you could issue a message for them to update and set the highest priority to that, so that no one would accidentally release something to develop that some user of the (old) production build would get?

**Comment by vytick** [comment]
> Do you have any plan on how to deal with app versions that have already been released to production and are looking at develop message-system config? Maybe after some time you could issue a message for them to update and set the highest priority to that, so that no one would accidentally release something to develop that some user of the (old) production build would get?

Message system was commented in the previously released versions, so no need to handle that, fortunately.

**Review by matejkriz — APPROVED** [review]
👍 approval for the desktop part
